### DISCUSSION TITLE
.qmd File Not Recognizing Python Syntax in VS CodeYour question in one sentence (e.g., How do I preview documents with Quarto?)

### DISCUSSION THREAD

**Discussion by cvianavinci** [quarto-dev/quarto-cli#10473] [Q&A] [answered]
### Description

![print](https://github.com/user-attachments/assets/34e12ec2-ceb8-4a92-a6b1-2a1cf0695470)

**Comment by mcanouil** [comment]
See
- quarto-dev/quarto#501

**Comment by cvianavinci** [comment]
What do I need to do with the quarto.tmLanguage file? Do I need to create an extension and include this file? I'm a bit of a beginner with QMD, could you help me?

**Reply by mcanouil** [reply, marked as answer]
You can't do anything.
That's a change that needs to be made on the Quarto VSCode extension.
A PR is already up.
So you simply have to be patient.

**Reply by cvianavinci** [reply]
Ok, thank you very much.
